build.sbt

```diff
@@ -6,3 +6,14 @@ scalaVersion := "2.13.5"
 
 libraryDependencies += "org.scalactic" %% "scalactic" % "3.2.5" % "test"
 libraryDependencies += "org.scalatest" %% "scalatest" % "3.2.5" % "test"
+
+libraryDependencies += "dev.zio" %% "zio" % "1.0.8"
+libraryDependencies += "dev.zio" %% "zio-streams" % "1.0.8"
+libraryDependencies +=  "dev.zio" %% "zio-test" % "1.0.8"
+
+
+//libraryDependencies ++= Seq(
```

Comment:
in case you are curious, this is how you can specify it as zioVersion:
```val zioVersion= "1.0.9"```

```libraryDependencies  += "dev.zio" %% "zio-test-sbt" % zioVersion % "test"```